db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
+          description: Dont scan image-inspectors image
+          modifier: deny
+          expression: !ruby/object:MiqExpression
+            exp:
+              ENDS WITH:
+                field: ContainerImage-name
+                value: "image-inspector"
+            context_type:
+          towhat: ContainerImage
+          file_mtime:
+          guid: e744245a-3d08-11e6-8d39-02422087d789
```

Comment:
@simon3z This doesn't seem possible. IIUC we can only filter with the fields of the model which are (from rails console):
```
 => ContainerImage(id: integer, tag: string, name: string, image_ref: string, container_image_registry_id: integer, ems_id: integer, last_sync_on: datetime, last_scan_attempt_on: datetime, digest: string, registered_on: datetime, region_number: integer, region_description: string, last_compliance_status: boolean, last_compliance_timestamp: datetime, display_registry: string) 
```